src/components/InstallUnsupportedMobile.tsx

```diff
@@ -58,8 +58,8 @@ export const InstallUnsupportedMobile: FunctionComponent<
             </Heading>
             <Text color="text.alternative" textAlign="center" marginTop={4}>
               <Trans>
-                MetaMask Snaps is in open beta and only supported via our
-                extension clients on desktop such as{' '}
+                MetaMask Snaps are currently only supported via our extension
```

Comment:
```suggestion
                MetaMask Snaps is currently only supported via our extension
```

I think it should be `is` as we're referring to the (singular) platform